### PR TITLE
Refactor console for 3.0

### PR DESCRIPTION
**Pull request by farost**
## Usage and product changes
We refactor console to support the updated `typedb` of 3.0-alpha.

## Implementation
Non-existing features are deleted. Non-implemented features are commented out (cloud, user management, options). 

The new output looks like this: 
```
> database create its-my-pr
Database 'its-my-pr' created
> transaction its-my-pr schema
its-my-pr::schema> define
                   attribute name, value string;
                   entity person, owns name;
                   
Success
its-my-pr::schema> commit
Transaction changes committed
> transaction its-my-pr write
its-my-pr::write> insert $x isa person, has name "John"; $y isa person, has name "Alice";
                  
Finished validation and compilation...
Finished writes. Streaming answers...

   ---------
    $x | iid 0x1e00000000000000000000 isa person
    $y | iid 0x1e00000000000000000001 isa person
   ---------

Finished. Total answers: 1
its-my-pr::write> commit
Transaction changes committed
> transaction its-my-pr read
its-my-pr::read> match $x has name $a;
                 
Finished validation and compilation...
Streaming answers...

   ---------
    $a | Alice isa name
    $x | iid 0x1e00000000000000000001 isa person
   ---------
    $a | John isa name
    $x | iid 0x1e00000000000000000000 isa person
   ---------

Finished. Total answers: 2
```

The release pipeline should be the same. However, we may need to update the dependencies on typedb artifacts later in the following smaller prs.

### CODE REVIEW

**Comment by flyingsilverfin** [comment]
Can we bring back the `$` before the variables on the left side
Also can we make sure there's a newline after the committed message (your example has it sometimes):
```
its-my-pr::schema> commit
Transaction changes committed
> transaction its-my-pr write
```

And maybe we can use `Finished validation and compilation...` instead of `Completed` since then we have Finished throughout?

File: README.md [hunk 1]
```diff
@@ -73,30 +62,30 @@ Console also offers command completion, accessible with a `tab` keypress.
 
 - `<query>` : Once you're in the transaction REPL, the terminal immediately accepts a multi-line TypeQL query, and will execute it when you hit enter twice. For example:
   ```
-  my-typedb-database::schema::write> define
-                                     name sub attribute, value string;
-                                     person sub entity, owns name;
+  my-typedb-database::schema> define
+                              attribute name, value string;
```
Comment (flyingsilverfin) [inline README.md:66]: nice catch

**Review by flyingsilverfin — APPROVED** [review]
Approved pending formatting changes in the previous comment!

**Comment by farost** [comment]
@flyingsilverfin fixed formatting in the description, I've inserted some excessive newlines while copying the text. Added `$` and two more `--` in the table dashes for symmetry.

**Pull request merged**